matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt

```diff
@@ -338,4 +341,26 @@ internal object RealmSessionStoreMigration : RealmMigration {
         realm.schema.get("EventInsertEntity")
                 ?.addField(EventInsertEntityFields.CAN_BE_PROCESSED, Boolean::class.java)
     }
+
+    private fun migrateTo18(realm: DynamicRealm) {
+        Timber.d("Step 17 -> 18")
+        realm.schema.create("UserPresenceEntity")?.apply {
```

Comment:
why not using the same format than the other migration?

Reply:
I think the code is clearer without so many:
```
?.
?.
?.
```
However, I will change it to be consistent with the other migrations